src/components/canvas-tools/templates/Circle.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <p>Outline colour</p>
+    <v-color-picker
+      v-model="circleColour"
+      :swatches="swatches"
+      mode="hexa"
+      show-swatches
+      hide-canvas
+      flat
+      hide-inputs
+      hide-mode-switch
+    />
+    <v-slider
+      prepend-icon="fa-ruler-vertical"
+      :hint="circleSizeHint"
+      v-model="circleSize"
+      :max="6"
+      :step="1"
+      ticks="always"
+    />
+    <p>Fill colour</p>
+    <v-color-picker
+      v-model="circleOutlineColour"
+      :swatches="swatches"
+      mode="hexa"
+      show-swatches
+      hide-canvas
+      flat
+      hide-inputs
+      hide-mode-switch
+    />
+    <v-radio-group v-model="circleStrokeStyle" row>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="strokeStyle" on-icon="fa-minus" off-icon="fa-minus" :value="0"></v-radio>
+      <v-spacer></v-spacer>
+      <v-radio class="mr-0" name="strokeStyle" on-icon="fa-ellipsis-h" off-icon="fa-ellipsis-h" :value="1"></v-radio>
+      <v-spacer></v-spacer>
+    </v-radio-group>
+    <v-container>
+      <v-row>
+        <v-spacer></v-spacer>
+        <v-switch
+          v-model="circleTemporarily"
+          :label="circleTemporarily === true ? 'Temporarily' : 'Permanent'"
+        ></v-switch>
+        <v-spacer></v-spacer>
+        <v-switch
+          v-model="circleShowRadius"
+          :label="circleShowRadius === true ? 'Show radius' : 'Hide radius'"
+        ></v-switch>
+        <v-spacer></v-spacer>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { Tool } from '@/tools/Tool'
+import { namespace } from 'vuex-class'
+import { Namespaces } from '@/store'
+import { ToolGetters, ToolsAction } from '@/store/modules/tools'
+
+const Tools = namespace(Namespaces.TOOLS)
+
+@Component({
+  name: 'DrawCircle',
+  computed: {}
+})
+export default class PopoutButton extends Vue {
+  @Tools.Getter(ToolGetters.TOOL) findTool!: (name: string) => Tool
+  @Tools.Action(ToolsAction.SET_COLOUR) setColour!: (colour: string) => void
+  @Tools.Action(ToolsAction.SET_SIZE) setSize!: (size: number) => void
+  @Tools.Action(ToolsAction.SET_SHOWRADIUS) setShowRadius!: (showRadius: boolean) => void
+  @Tools.Action(ToolsAction.SET_OUTLINECOLOUR) setOutlineColour!: (outlineColour: string) => void
```

Comment:
missing underscores here for all these @Tools. SET_OUTLINE_COLOUR then correctly sets the function name as setOutlineColour.